recipes/cmd2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{%set name = "cmd2" %}
+{%set version = "0.6.8" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "ac780d8c31fc107bf6b4edcbcea711de4ff776d59d89bb167f8819d2d83764a8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  # patches:
+  #   - python35-compat.patch  # [py35]
+
+build:
+  skip: True  # [py35]
```

Comment:
This is fine. Just open an issue in the feedstock to remind you to remove the `Skip` once you patch or update.